frontend/src/app/settings/core/configuration-settings/configuration-settings.component.html

```diff
@@ -0,0 +1,15 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+<div id="editor" class="h-[70vh] min-h-[200px]"></div>
+<div class="m-2 flex gap-2">
+  <button mat-stroked-button (click)="submitValue()" color="primary">
+    <span>Submit</span>
```

Comment:
```suggestion
    <span>Save</span>
```

Reply:
Done 👍 
Also changed the icon to the "save" icon.